scripts/variantstore/scripts/merge_and_rescore_vdses.py

```diff
@@ -239,5 +250,8 @@ def patch_variant_data(vd: hl.MatrixTable, site_filters: hl.Table, vets_filters:
 
     rescored_vd = patch_variant_data(vd, site, vets)
     merged_and_rescored_vds = hl.vds.VariantDataset(tmp_merged_vds.reference_data, rescored_vd)
+    if samples_to_remove_table:
+        merged_and_rescored_vds = hl.vds.filter_samples(merged_and_rescored_vds, samples_to_remove_table, keep=False,
```

Comment:
i dont know how robust `filter_samples` is but computationally would it be better to drop samples before you process the data?  (if `filter_samples` fails if you pass it a sample name that doesnt exist then i retract my statement)

Reply:
Hmm I can certainly try dropping samples after the merge but before the rescoring... I'll try that.